.github/workflows/release-trigger-builds.yml

```diff
@@ -0,0 +1,74 @@
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Install Github CLI
+        run: |
+          echo "Installing GH CLI."
+          if ! command -v gh &> /dev/null; then
+            echo "Installing GH CLI."
+            type -p curl >/dev/null || (sudo apt-get update && sudo apt-get install curl -y)
+            curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg
+            sudo chmod go+r /usr/share/keyrings/githubcli-archive-keyring.gpg
+            echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null
+            sudo apt-get update
+            sudo apt-get install gh -y
+          else
+            echo "GH CLI is already installed."
+          fi
+        shell: bash
+
+      - name: Trigger selected branches
+        run: |
+          # VARS
+          # names defined in workflow files per components
+          COMPONENTS_OSS=("awie" "dsm" "gorgone" "ha" "open-tickets" "web")
+          COMPONENTS_OSS_FULL=("awie" "dsm" "gorgone" "ha" "open-tickets" "web" "perl-cpan-libraries" "perl-crypt-argon2" "perl-filesys-smbclient" "perl-json-path" "perl-keepass-reader" "perl-libssh-session" "perl-net-curl" "plink")
+          COMPONENTS_MODULES=("anomaly-detection" "autodiscovery" "bam" "cloud-business-extensions" "cloud-extensions" "it-edition-extensions" "lm" "map" "mbi" "ppm")
+          COMPONENTS_MOULES_FULL=("anomaly-detection" "autodiscovery" "bam" "cloud-business-extensions" "cloud-extensions" "it-edition-extensions" "lm" "map" "mbi" "ppm" "php-pecl-gnupg" "sourceguardian-loader")
+          COMPONENTS_COLLECT=("Centreon collect")
+          RUNS_URL=""
+
+          if [[ ${{ inputs.dispatch_target_release_branch }} =~ "^release-2[0-9][0-9]-next$" ]];then
+            echo "Using ${{ inputs.dispatch_target_release_branch }} as release branch to build testing packages."
+            RUNS_URL="https://github.com/centreon/centreon-modules/actions?query=branch%3A${{ inputs.dispatch_target_release_branch }}"
+          else
+            echo "Invalid release branch name, please check the release branch name."
+          fi
+
+          if [[ "${{ inputs.dispatch_content }}" == "FULL" ]]; then
+            echo "Requested ${{ inputs.dispatch_content }} content, triggering all component workflows."
+            for COMPONENT in ${COMPONENTS_OSS_FULL};do
+              gh workflow run $COMPONENT -r ${{ inputs.dispatch_target_release_branch }}
+            done
+          else
+            echo "Requested ${{ inputs.dispatch_content }} content, triggering centreon named components only."
+            for COMPONENT in ${COMPONENTS_OSS};do
+              gh workflow run $COMPONENT -r ${{ inputs.dispatch_target_release_branch }}
+            done
+          fi
+
+          if [[ $? -eq 0 ]];then
```

Comment:
I think this check is not necessary because the job will fail on the error command

Reply:
indeed, just displaying the success case should be enough here